### PR TITLE
Add postinstall, deploy-staging, deploy-production scripts

### PR DESCRIPTION
**Pull request by njsfield**
Related #124

### CODE REVIEW

**Review by emilyb7 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -34,21 +33,20 @@
...
-    "nodemon": "^1.11.0",
     "postcss-clean": "^1.0.2",
     "postcss-cli": "^2.6.0",
     "postcss-custom-media": "^5.0.1",
     "postcss-custom-properties": "^5.0.2",
     "postcss-import": "^9.1.0"
+  },
+  "devDependencies": {
+    "eslint": "^3.19.0",
+    "keystone-nightwatch-e2e": "^0.2.14",
```
Comment (des-des) [inline package.json]: do we use this?

File: package.json [hunk 2]
```diff
@@ -34,21 +33,20 @@
...
+    "nightwatch": "^0.9.16",
+    "nodemon": "^1.11.0",
+    "saucelabs": "^1.4.0",
+    "selenium-download": "^2.0.10"
```
Comment (des-des) [inline package.json]: do we use this?

File: package.json [hunk 3]
```diff
@@ -34,21 +33,20 @@
...
     "postcss-custom-media": "^5.0.1",
     "postcss-custom-properties": "^5.0.2",
     "postcss-import": "^9.1.0"
+  },
+  "devDependencies": {
+    "eslint": "^3.19.0",
+    "keystone-nightwatch-e2e": "^0.2.14",
+    "nightwatch": "^0.9.16",
+    "nodemon": "^1.11.0",
+    "saucelabs": "^1.4.0",
```
Comment (des-des) [inline package.json]: this?

**Review by des-des — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (njsfield) [reply package.json]: We do

File: package.json (hunk 3 above)
Comment (njsfield) [reply package.json]: Not currently so will remove

File: package.json (hunk 1 above)
Comment (njsfield) [reply package.json]: We don't, will remove

**Comment by njsfield** [comment]
@emilyb7 reassigning for review for approval after changes, then reassigning to @des-des

**Review by emilyb7 — APPROVED** [review]

**Comment by njsfield** [comment]
@des-des With postinstall script set up, our heroku build fails;
image
Heroku cant seem to locate postcss. I've included it in the package dependencies, but heroku still fails. Any thoughts?

**Comment by des-des** [comment]
@njsfield @emilyb7 

The problem is caused by npm5 I think. Trying to run `NODE_ENV=production npm i` on a fresh clone with node7, npm5 will fail locally.

I expect the easiest solution is to drop down to npm 4 (and delete package.lock which causes heroku to use npm 5)

**Comment by emilyb7** [comment]
@des-des deleting package-lock seems to have worked. I didn't have to drop down the npm version. 

Have also upgraded npm deploy scripts to force push.

**Review by des-des — APPROVED** [review]

**Pull request merged**